cloudnet-modules/cloudnet-bridge/src/main/java/de/dytanic/cloudnet/ext/bridge/BridgeConfiguration.java

```diff
@@ -23,15 +23,20 @@
     private Collection<ProxyFallbackConfiguration> bungeeFallbackConfigurations;
     private Map<String, String> messages;
     private boolean logPlayerConnections = true;
+    private boolean hubCommandEnabled = true;
+    private ArrayList<String> hubCommandAliases;
```

Comment:
Please use the Collection interface instead of an ArrayList here

Reply:
List*

Reply:
Why not a Collection?

Reply:
Because the order is important (to get the name the command should have)

Reply:
The name doesn't matter, and Velocity doesn't even differentiate between the name and aliases.